src/jazkarta/easyformplugin/savecontent/profiles/default/actions.xml

```diff
@@ -19,5 +19,21 @@
       </property>
       <property name="visible">True</property>
     </object>
+    <object meta_type="CMF Action"
```

Comment:
Maybe this should go in `profiles/default/types/jazkarta.efp.data_folder.xml` with `category="document_actions"` and no condition needed, but having this action on the form itself, maybe isn't bad either.